x-pack/plugins/security_solution/public/overview/containers/overview_host/index.tsx

```diff
@@ -38,6 +40,13 @@ export interface OverviewHostProps extends QueryTemplateProps {
 
 const OverviewHostComponentQuery = React.memo<OverviewHostProps & PropsFromRedux>(
   ({ id = ID, children, filterQuery, isInspected, sourceId, startDate, endDate }) => {
+    const { activeSourceGroupId, getManageSourceGroupById } = useManageSource();
```

Comment:
`activeSourceGroupId` will be determined by which page/view the component is called in